website/docs/guides/requests/1-create-ttl-index/create-ttl-index.js

```diff
@@ -0,0 +1,4 @@
+db.runCommand({
```

Comment:
Why was `website/docs/guides/requests/1-create-ttl-index.js` renamed to `website/docs/guides/requests/1-create-ttl-index/create-ttl-index.js` with extra folder in it? :thinking:

Reply:
It would make sense to have three test suites (for TTL indexes, for full-text search, for vector search) with multiple test cases named after existing files: "1-create-ttl-index", etc.

Reply:
@AlekSi Sure, although not sure what should be the directory name in that case. What's the motivation behind numbers in testsuite names ("1-*")?